angular/src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,42 @@
+div.container.valign-wrapper {
+    width: 100%;
+    height: 50px;
+    display: flex;
+    justify-content: space-around;
+    background-color: #ffe8dc;
+    margin: 0%;
```

Comment:
When you write `0` in CSS, the unit is always optional. For example, writing `margin: 0;` is legal (but `margin: 10;` is not).